SPEC.md

```diff
@@ -495,6 +495,9 @@ The ACE must provide a Metadata service on the address given to the applications
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
 
+**NOTE**: Implementations can insert an authorization token into AC_METADATA_URL.
+For example, AC_METADATA_URL can be set to `https://10.0.0.1:8888/Y4vFeVZzKM2T9rwkpWHfqXuGsNjS6O5c` with the path portion acting as a token to uniquely and securely identify a pod.
```

Comment:
I guess. To be honest, I still feel uncomfortable with specifying level of entropy. The level of security is something that the implementation should decide on its own. It's immaterial for interop and prescribing something that's not actually required always makes me uneasy.